docs/IKOS.md

```diff
@@ -0,0 +1,61 @@
+
+# IKOS Integration
+
+IKOS is a static analysis tool that performs interprocedural analysis on C/C++ code.
+
+Make sure you are in the Space ROS Docker container before running the following commands.
+
+## Preparing Space ROS sources
+
+A manifest of the exact sources of Space ROS used to produce the current image is saved as `spaceros.repos` in the `/opt/spaceros` directory.
+To clone all sources from this manifest you can use the command sequence
+
+```bash
+spaceros-user@d10d85c68f0e:~/$ mkdir -p spaceros_ws/src
+spaceros-user@d10d85c68f0e:~/spaceros_ws$ cd spaceros_ws
+spaceros-user@d10d85c68f0e:~/spaceros_ws$ vcs import src < /opt/spaceros/scripts/spaceros.repos
+```
+
+## Running an IKOS Scan
+
+IKOS uses special compiler and linker settings in order to instrument and analyze binaries.
+To run an IKOS scan on all of the Space ROS test binaries (which will take a very long time), run the following command at the root of the Space ROS workspace:
+
+```bash
+spaceros-user@d10d85c68f0e:~/spaceros_ws$ CC="ikos-scan-cc" CXX="ikos-scan-c++" LD="ikos-scan-cc" colcon build --cmake-args -DSECURITY=ON -DINSTALL_EXAMPLES=OFF -DCMAKE_EXPORT_COMPILE_COMMANDS=ON --no-warn-unused-cli
```

Comment:
Honestly, I didn't test this part since IKOS is new to me. If it builds upon colcon, then yes, we will need merge-install. Also, `/opt/spaceros` is owned by `spaceros-user`

Reply:
I think IKOS is open to both merge-install or the standard one. It comes to the preference where the new builds are done with merge install or not. We could probably skip if needed.